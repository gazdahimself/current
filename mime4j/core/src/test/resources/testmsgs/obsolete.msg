Subject :The obsolete syntax allow spaces before the colon
       
 and also empty lines in folding.
Date    :     Malformed Date.
Invalìd-Header: this is not valid.
HeaderWithWSP 	  	:		value.

body
