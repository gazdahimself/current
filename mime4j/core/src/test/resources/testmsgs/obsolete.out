Subject :The obsolete syntax allow spaces before the colon
       
 and also empty lines in folding.
Date    :     Malformed Date.
HeaderWithWSP 	  	:		value.

body
